src/routes/blog/post/appwrite-vs-cloudinary/+page.markdoc

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "Appwrite Storage: open-source alternative to Cloudinary"
+description: Get a comprehensive comparison between Appwrite and Cloudinary to decide which platform fits your media and files needs best.
+date: 2024-09-03
+cover: /images/blog/appwrite-compared-to-supabase/cover.png
```

Comment:
@SnezhannaM is this correct?

Reply:
@adityaoberai yes, all competitor blogs have the same cover

Reply:
@SnezhannaM should we update the name then? it feels a bit weird to have this naming. @adityaoberai thoughts?

Reply:
@adityaoberai I changed the name and updated this blog and the Supabase one.